docs/getting_started/triton.rst

```diff
@@ -0,0 +1,81 @@
+Triton
+======
+
+.. versionadded:: 1.1
+
+As of Mantl 1.1 you can bring a Mantl cluster up on Joyent's Triton. Please be
+sure to use at least Terraform 0.6.15: the first version with the required
+resources.
+
+Configuring Triton for Terraform
+--------------------------------
+
+Before we can build any servers using Terraform and Ansible, we need to
+configure authentication. We'll be filling in the authentication variables for
+the template located at ``terraform/triton.sample.tf``. The beginning of it
+looks like this:
+
+.. code-block:: json
+
+   # this sample assumes that you have `SDC_ACCOUNT`, `SDC_KEY_MATERIAL`,
+   # `SDC_KEY_ID`, and `SDC_URL` in your environment from (for example) using the
+   # Triton command-line utilities. If you don't, set `account`, `key_material`,
+   # `key_id`, and `url` in the provider below
+   provider "triton" {}
+
+   variable key_path { default = "~/.ssh/id_rsa.pub" }
+
+Copy ``terraform/triton.sample.tf`` in it's entierty to the root of the project
+as ``triton.tf`` to start customization. In the next section, we'll explain how
+to obtain the settings mentioned above.
+
+Basic Settings
+^^^^^^^^^^^^^^
+
+First, we'll need an account. This is the username you use to log into Triton.
```

Comment:
Possibly link to https://docs.joyent.com/public-cloud/getting-started to give people direction about how to create an account?